### PR TITLE
Added authorization header

### PR DESCRIPTION
**Pull request by furebo**
# Description

This PR contains changes that allow user to view and edit his profile information

## Type of change

Please check the options that are relevant.

- [ X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would as expected)
- [ ] This change requires a documentation update

# How should this be manually tested or reproduced?

- Clone this repo
- Checkout to `bug-fix-userprofile`
- Run the command `npm i`
- Run the command `npm run test` to run tests locally 
- If you push new changes, the test shall run on travis

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I have added tests that prove that my feature works
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by SimonImanigirimpuhwe — CHANGES_REQUESTED** [review]
Fix that and while updating information don't allow user to update email since it doesn't even get updated on the backend

Screen Shot 2021-02-22 at 11 20 06

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "space-bn-frontend",
```
Comment (leny62) [inline package.json:2]: Kindly rebase!

**Review by leny62 — CHANGES_REQUESTED** [review]
Kindly respond to @SimonImanigirimpuhwe 's requested changes, and rebase to make this package.json file uptodate with the develop branch.

**Review by SimonImanigirimpuhwe — APPROVED** [review]
Good improvement

File: src/components/ProfileInfos/index.js [hunk 2]
```diff
@@ -299,68 +306,76 @@ const ProfileInfos = () => {
                                 <Grid item xs>
                                     <form className={classes.form}>
                                         <div> 
-                                                                                    
+                                            <Checkbox color='primary' className={classes.checkboxField} />                                        
                                             <LabelWrapper> Firstname:</LabelWrapper>
-                                            <InputWrapper>
```
Comment (SamuelSabasNayo) [inline src/components/ProfileInfos/index.js:304]: You can make this input visible.
Screen Shot 2021-02-23 at 16 19 20

Comment (furebo) [reply src/components/ProfileInfos/index.js:304]: I think while this is not editing page we don't need to make them visible

**Review by leny62 — APPROVED** [review]

File: src/components/ProfileInfos/index.js (hunk 2 above)
Comment (SimonImanigirimpuhwe) [reply src/components/ProfileInfos/index.js:304]: There is no inputs there @SamuelSabasNayo , that is the page of only displaying info

**Review by thepatrickniyo — APPROVED** [review]

**Pull request merged**